### ISSUE TITLE
Artifactory Generic Deploy Build Task not working with file spec 404

### ISSUE THREAD

**Issue by tobmat** [jfrog/bamboo-artifactory-plugin#175]
**Describe the bug**
Artifactory Generic Deploy Build Task not working with file spec but is working with the 'Legacy Patterns'.  It gets a 404 using the file spec option.  The file spec option is working on the deploy task side with identical file spec.

**To Reproduce**
Attempt to push any file using file spec on 'Artifactory Generic Deploy Build Task'

**Expected behavior**
File would be pushed to artifactory.  

**Versions**
- Bamboo Artifactory Plugin version: 3.2.8
- Bamboo version 8.0.4 build 80012
- Artifactory version 7.35.2

**Comment by lbgo** [comment]
Hi @tobmat,

Did you solved this issue?

Thanks.

**Comment by tobmat** [comment]
Hey @lbgo - Somehow I missed seeing this.  Sorry for the delayed response.  We didn't resolve this and continue to use the 'Legacy Patterns'.

**Comment by tobmat** [comment]
I don't think this issue was every actually addressed.  We are still using the Legacy Patterns.
